Resources/Textures/_DV/Clothing/Head/Helmets/eva_epistemics.rsi/meta.json

```
@@ -0,0 +1,38 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Sprite edited by Flareguy for SS14, original unedited sprite can be found in https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a | vulpkanin version taken from Paradise station at https://github.com/ParadiseSS13/Paradise/commit/f0fa4e1fd809482fbc104a310aa34cebf7df157d modified by Floofers",
```

tl;dr Where did *you* get the sprites from? I was unable to understand that from these copyright statements. I'd be happy to help you rewrite this, just need the explanation on where these came from.

I'm assuming from this copyright statement that:
1. there were some original sprites (tgstation, Paradise)
(note, that tgstation link is just... wrong, it does not link to any sprites. But I'm assuming you didn't write that link yourself)
2. then, some non-DeltaV people (Flareguy, Floofers) made some modifications for some other repos
3. then, we're taking those modified versions for DeltaV?

If that's true, we should link the repos that contain the modified versions AND the originals that you've already linked here. You're missing a link for step 2 in the chain.

---

I'm honestly not sure if I really even need to include the "sprite edited" part because I've basically gone and made my own suit sprite. It was only very loosely based off of the base EVA suit. Liscencing is not my strong suit, so apologies for that.

---

Ohhh I see, you copied the copyright from the default suit. Ok that makes more sense.

I would also say licensing is not my strong suit, as would most I think haha

Great! If this is your original work then that makes it easy. Just add your own name.

```suggestion
    "copyright": "Sprites by IAmNotGray. Loosely derived from eva.rsi, which has the following copyright: Sprite edited by Flareguy for SS14, original unedited sprite can be found in https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a | vulpkanin version taken from Paradise station at https://github.com/ParadiseSS13/Paradise/commit/f0fa4e1fd809482fbc104a310aa34cebf7df157d modified by Floofers",
```

Or something like that. 

If you don't want to add your name for some reason, just say "Sprites by an anonymous DeltaV contributor." or something

---

Feel free to change the name too, of course (if IAmNotGray is not the version you want to use)

---

I would kind of agree that your sprites are basically completely new artwork, but I suppose the general shape is roughly the same as the base one, so it's better to be safe than sorry. Rest assured that anyone actually reading this copyright will also very clearly see that you've done most of the creative work on these!

---

I'll make the changes to those lines, then. Thanks a ton for the help.